### PR TITLE
daps: add page

### PR DESCRIPTION
**Pull request by quatran**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
3.1

### CODE REVIEW

File: pages/common/daps.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# daps
+
+> DAPS is an open source program for transforming DocBook XML into output formats such as HTML or PDF.
+> More information: <https://opensuse.github.io/daps/doc/index.html>.
+
+- Check your DocBook XML file is vaild:
```
Comment (marchersimon) [inline pages/common/daps.md:6]:
```suggestion
- Check if a DocBook XML file is vaild:
```

File: pages/common/daps.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`daps -d {{path/to/docbook.xml}} validate`
+
+- Convert your XML file into PDF:
```
Comment (marchersimon) [inline pages/common/daps.md:10]:
```suggestion
- Convert a DocBook XML file into PDF:
```

File: pages/common/daps.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`daps -d {{path/to/DC-file}} pdf`
+
+- Convert your XML file into HTML:
```
Comment (marchersimon) [inline pages/common/daps.md:14]:
```suggestion
- Convert a DocBook XML file into a single HTML file:
```

File: pages/common/daps.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+> DAPS is an open source program for transforming DocBook XML into output formats such as HTML or PDF.
+> More information: <https://opensuse.github.io/daps/doc/index.html>.
+
+- Check your DocBook XML file is vaild:
+
+`daps -d {{path/to/docbook.xml}} validate`
+
+- Convert your XML file into PDF:
+
+`daps -d {{path/to/DC-file}} pdf`
```
Comment (marchersimon) [inline pages/common/daps.md:12]:
```suggestion
`daps -d {{path/to/docbook.xml}} pdf`
```

File: pages/common/daps.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Convert your XML file into HTML:
+
+`daps -d {{path/to/DC-file}} html --single`
```
Comment (marchersimon) [inline pages/common/daps.md:16]:
```suggestion
`daps -d {{path/to/docbook.xml}} html --single`
```

File: pages/common/daps.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display help:
+
+`daps --help`
+
+- Display version:
```
Comment (marchersimon) [inline pages/common/daps.md:22]:
```suggestion
- Display version information:
```

File: pages/common/daps.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/daps.md:16]:
```suggestion
`daps -d {{path/to/file.xml}} html --single`
```

File: pages/common/daps.md (hunk 4 above)
Comment (navarroaxel) [reply pages/common/daps.md:12]:
```suggestion
`daps -d {{path/to/file.xml}} pdf`
```

File: pages/common/daps.md (hunk 6 above)
Comment (navarroaxel) [reply pages/common/daps.md:22]: Every output is information :thinking:

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @quatran

**Pull request merged**